docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -535,6 +535,27 @@ Each organism object has the following fields:
             <td></td>
             <td>Metadata fields associated with the organism.</td>
         </tr>
+        <tr>
+            <td>`earliestReleaseDate`</td>
+            <td>Object</td>
+            <td></td>
+            <td>Configuration object for enabling and configuring the `earliestReleaseDate` metadata field.</td>
```

Comment:
Added [`e51ae77` (#3380)](https://github.com/loculus-project/loculus/pull/3380/commits/e51ae77190899d895e865d0f2c84c38335c89f1b) [`14cba2e` (#3380)](https://github.com/loculus-project/loculus/pull/3380/commits/14cba2ed08a365027051bf25c99af8d0a3836cd7)

(In the long term I think it would be good to have this in a more dedicated [explanation](https://diataxis.fr/explanation/) section -- this is supposed to be only [reference](https://diataxis.fr/reference/) information I think)

Reply:
Agreed, having an explanation section would be great! (Feel free to introduce one whenever you like.)